### ISSUE TITLE
_base_manager attribute not set for proxy models

### ISSUE THREAD

**Issue by vittoriozamboni** [django-mptt/django-mptt#368]
Hi! First of all, thank you for this awesome work.
When I try to upgrade to 0.7.1 from 0.6.1 my tests break with Proxy models: 

```
File "/Users/vittorio/env/django-groups-manager/lib/python2.7/site-packages/mptt/managers.py", line 465, in insert_node
    if self._base_manager:
AttributeError: 'ProjectManager' object has no attribute '_base_manager'
``` 

My model and it's custom manager are defined as follows:
```python
class ProjectManager(TreeManager):

    def all(self, *args, **kwargs):
        queryset = super(ProjectManager, self).all(*args, **kwargs)
        return queryset.filter(group_type__codename='project')

    def filter(self, *args, **kwargs):
        queryset = super(ProjectManager, self).filter(*args, **kwargs)
        return queryset.filter(group_type__codename='project')


class Project(Group):
    objects = ProjectManager()

    class Meta:
        proxy = True
```

If I add the two *init* values, it works with proxy and non proxy models:
```python
    def contribute_to_class(self, model, name):
        super(TreeManager, self).contribute_to_class(model, name)

        self._base_manager = None
        self.tree_model = model

        if not (model._meta.abstract or model._meta.proxy): 
```
Am I missing something?

**Comment by craigds** [comment]
@vittoriozamboni thanks for reporting this. Can you please check this fix and make sure it fixes the issue? Cheers

**Comment by vittoriozamboni** [comment]
Yes, it works!
Thank you @craigds !

**Comment by craigds** [comment]
Thanks for confirming that :) Released this as 0.7.2
